sdk/core/azure-core/azure/core/tracing/_abstract_span.py

```diff
@@ -26,23 +20,17 @@
 HttpResponseType = Union[HttpResponse, AsyncHttpResponse, RestHttpResponse, AsyncRestHttpResponse]
 HttpRequestType = Union[HttpRequest, RestHttpRequest]
 
-if TYPE_CHECKING:
-    AttributeValue = Union[
-        str,
-        bool,
-        int,
-        float,
-        Sequence[str],
-        Sequence[bool],
-        Sequence[int],
-        Sequence[float],
-    ]
-    Attributes = Optional[Dict[str, AttributeValue]]
-
-try:
-    from typing_extensions import Protocol
-except ImportError:
-    Protocol = object  # type: ignore
+AttributeValue = Union[
+    str,
+    bool,
+    int,
+    float,
+    Sequence[str],
+    Sequence[bool],
+    Sequence[int],
+    Sequence[float],
+]
+Attributes = Optional[Dict[str, AttributeValue]]
```

Comment:
This way we can avoid the double Optional for Attributes below
```suggestion
Attributes = Dict[str, AttributeValue]
```